_docs/resources/engaging-helix.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Engaging Helix
+layout: docs
+status: stable
+resource: true
+category: Resources
+last-modified: 2018-07-11
+usage: >
+   The long and short of it is that you should reach out the the Helix team anytime the team might be helpful to you, but it’s not always easy to know what that is. Use the following scenarios as a guide for when you should contact the Helix team.
+---
+
+## When to engage
+
+<section class="static-section" markdown="1">
+
+### Feedback & review
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-4" %}
+
+![When I want to know if I’m using Helix correctly or not]({{site.url}}/assets/images/resources/engaging-helix/feedback-and-review-using-helix-correctly.png){: width="100%"}
```

Comment:
Agree with Laura's comments about the supporting text occupying a weird place in the info hierarchy. Maybe we can add the text into the image? Like 
<img width="525" alt="screen shot 2018-07-13 at 3 40 20 pm" src="https://user-images.githubusercontent.com/5124722/42712922-19c27c06-86b3-11e8-9fc5-9c095ae63927.png">